HerPortal/Models/HomepageViewModel.cs

```diff
@@ -68,13 +68,13 @@ public CsvFile(CsvFileData csvFileData, string downloadLink)
         List<string> consortiumCodes
```

Comment:
probably worth going through a lot of these structures. similar to joe's comment above a lot of this code was written in a strange way due to the lack of the role, ie
- does a list of consortium codes need be passed here?
- is `DataAccessProvider.GetConsortiumCodesForUser` and its associated wrappers still needed? getting consortiums of users is now a much more trivial task